src/server/graphql/types/SubscriptionTypeEnum.js

```diff
@@ -0,0 +1,15 @@
+import {GraphQLEnumType} from 'graphql';
```

Comment:
not yet! i built it, decided not to use it for this case, but I think we'll use it in the future.

Reply:
Just pointing out that smells a bit [YAGNI-esque](https://testing.googleblog.com/2017/08/code-health-eliminate-yagni-smells.html) 😉

Reply:
i interpret yagni differently. It'd be something like "do i need to accept an array or a single value? well, just a single value, but i'll build it to accept an array because i might need it later". it's adding unnecessary complexity to something that doesn't need it yet, and may never need it. i'd never write the file just for fun, but since it's already here & we'll likely use it later, there's no harm in keeping it. Since this file isn't referenced by anything there, it's nothing more than a scratchpad that's kept with the repo. yay webpack!

Reply:
Good thoughts! I'm not advocating deleting it. I do however think it's worth it in the future to avoid the temptation to check in code which isn't quite yet used, for no other reason than it can be confusing to the new developer 😆